src/components/Pressable/GenericPressable/types.ts

```diff
@@ -19,7 +19,7 @@ type RequiredAccessibilityLabel =
            * @example 'Search'
            * @example 'Close'
            */
-          accessibilityLabel: string;
+          accessibilityLabel?: string;
```

Comment:
I think this should be a required prop @VickyStash

Reply:
@blazejkustra Updated, please take a look